From 9a35bef82a71c46470244d672f284e53ce6132cb Mon Sep 17 00:00:00 2001
From: Khai Wen Ng <khai.wen.ng@intel.com>
Date: Fri, 12 Sep 2025 03:33:03 +0800
Subject: [PATCH 55/61] media: i2c: maxim-serdes: max9296a: update register

Register does not show the reflect the status of first pipe.

Signed-off-by: Khai Wen Ng <khai.wen.ng@intel.com>
---
 drivers/media/i2c/maxim-serdes/max9296a.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/maxim-serdes/max9296a.c b/drivers/media/i2c/maxim-serdes/max9296a.c
index cfbf11adbb1ec..9c983cecb6d81 100644
--- a/drivers/media/i2c/maxim-serdes/max9296a.c
+++ b/drivers/media/i2c/maxim-serdes/max9296a.c
@@ -61,7 +61,7 @@
 #define MAX9296A_VIDEO_PIPE_SEL_STREAM(p)	(GENMASK(1, 0) << ((p) * 3))
 #define MAX9296A_VIDEO_PIPE_SEL_LINK(p)		BIT(2 + (p) * 3)
 
-#define MAX9296A_VPRBS(p)			(0x1fc + (p) * 0x20)
+#define MAX9296A_VPRBS(p)			(0x1dc + (p) * 0x20)
 #define MAX9296A_VPRBS_VIDEO_LOCK		BIT(0)
 #define MAX9296A_VPRBS_PATGEN_CLK_SRC		BIT(7)
 #define MAX9296A_VPRBS_PATGEN_CLK_SRC_150MHZ	0b0
-- 
2.50.1

